src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -170,17 +170,6 @@ public async Task<WorkerStatus> GetWorkerStatusAsync()
                 }
             }
 
-            // get the process stats for the worker
-            var workerProcessStats = _rpcWorkerProcess.GetStats();
-            workerStatus.ProcessStats = workerProcessStats;
-
-            if (workerProcessStats.CpuLoadHistory.Any())
-            {
-                string formattedLoadHistory = string.Join(",", workerProcessStats.CpuLoadHistory);
-                int executingFunctionCount = FunctionInputBuffers.Sum(p => p.Value.Count);
-                _workerChannelLogger.LogDebug($"[HostMonitor] Worker process stats: EffectiveCores={_environment.GetEffectiveCoresCount()}, ProcessId={_rpcWorkerProcess.Id}, ExecutingFunctions={executingFunctionCount}, CpuLoadHistory=({formattedLoadHistory}), AvgLoad={workerProcessStats.CpuLoadHistory.Average()}, MaxLoad={workerProcessStats.CpuLoadHistory.Max()}");
```

Comment:
As mentioned above, the event names and structure are the same. A new Host.Concurrency category was added, and the host CPU stats are now logged under that, whereas previously I believe they were logged under Host.Controllers.Host, because the logger is flowed down from the controller. So the category is changed, but structure is the same.